src/cogent3/maths/distance_transform.py

```diff
@@ -61,6 +63,7 @@
     argsort,
     array,
     asarray,
+    intersect1d,
```

Comment:
The LGTM service identified this as an unused import (see the "conversation" page for this PR).